README.md

```diff
@@ -30,10 +30,12 @@ Building
 
 You need to install both OpenResty and [PostgreSQL](http://www.postgresql.org/) 9.x, obviously.
 
-You need to install [pandoc](http://pandoc.org/) to your system. On Mac OS X, for example, it's as simple as
+You need to install [pandoc](http://pandoc.org/) and [marked](https://marked.js.org/) to your system.
```

Comment:
Do we still need pandoc at all? Seems like it was only used to generate the plain text version. Is marked also capable of producing such output?

Reply:
@agentzh marked can only convert markdown to html.